src/domain/relay/limit-addresses.mapper.ts

```diff
@@ -156,39 +156,33 @@ export class LimitAddressesMapper {
     value: bigint;
     data: `0x${string}`;
   } | null {
-    try {
-      const safeDecodedData = this.safeDecoder.decodeFunctionData({
-        data,
-      });
+    const safeDecodedData =
+      this.safeDecoder.decodeFunctionData.execTransaction(data);
 
-      if (safeDecodedData.functionName !== 'execTransaction') {
-        return null;
-      }
-
-      return {
-        to: safeDecodedData.args[0],
-        value: safeDecodedData.args[1],
-        data: safeDecodedData.args[2],
-      };
-    } catch {
+    if (!safeDecodedData) {
       return null;
```

Comment:
Nit: as before, should we log a warning here?

Reply:
Logging now occurs if decoding fails in 36b45c17e2e254e6365f838b1b14e566de930810.